Configs/Config.ps1

```diff
@@ -121,4 +121,5 @@ $config.vds = @{
         linkproto = [string]'LLDP'
         linkoperation = [string]'Both'
         mtu = [int]1500
+        uplinkCount = [int]8
```

Comment:
Ah, I missed the Keys part. Definitely need to make sure that the proper number of values are in the config, not just the contents of the values! I think I fixed now, and pushed the changes.